pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fme"
+version = "2024.10.3"
```

Comment:
This version number was just for testing things out on testpypi.  I'm thinking we should use the `2023.12.0` version that's tagged in the repo for actual pypi, but I wanted to check before actually committing a version.